app/assets/javascripts/mapknitter/Map.js

```diff
@@ -134,6 +134,7 @@ MapKnitter.Map = MapKnitter.Class.extend({
         }
       });
     });
+    if (map._imgGroup.getBounds().isValid()) map.fitBounds(map._imgGroup.getBounds());
```

Comment:
The issue is timing - placement of `map._imgGroup.getBounds().isValid()` late enough that it actually returns `true`. When booting in the browser, you can run it early and it's `false` but after the page loads, it's `true`, so we just need some event late enough to run this after all of that...